cpan/Scalar-List-Utils/ListUtil.xs

```diff
@@ -1971,19 +1973,24 @@ PPCODE:
         if (s > nameptr && *s == ':' && s[-1] == ':') {
             end = s - 1;
             begin = ++s;
+#if PERL_VERSION_LT(5, 41, 0)
             if (quotes_seen)
                 need_subst = TRUE;
+#endif
         }
+#if PERL_VERSION_LT(5, 41, 0)
```

Comment:
I wonder if the 0 is still valid? and elsewhere

Reply:
We won't know what it should be until it's approved and merged.